app_dart/lib/src/service/content_aware_hash_service.dart

```diff
@@ -112,4 +127,102 @@ interface class ContentAwareHashService {
     // Fail
     return null;
   }
+
+  /// Finds the hash status for [job] and updates any tracking docs.
+  Future<MergeQueueHashStatus> processWorkflowJob(WorkflowJobEvent job) async {
+    final hash = await hashFromWorkflowJobEvent(job);
+    if (hash == null) return MergeQueueHashStatus.unknown;
+
+    final headSha = job.workflowJob!.headSha!;
+
+    final r = const RetryOptions(
+      maxAttempts: 5, // number of entries in the merge group?
```

Comment:
In other places in app_dart we use 3 retries, so I'd be curious why we think we need 5 (or more)